### ISSUE TITLE
Use amp-base-carousel for amp-lightbox-gallery

### ISSUE THREAD

**Issue by sparhami** [ampproject/amphtml#21568]
There are a few ways `amp-lightbox-gallery` would benefit from switching over to `amp-base-carousel`. For example,  #21559, #21374 and #22608 would be fixed. Another improvement would be to allow quicker swiping between slides,

This should be done with an experiment flag just to make sure there is no unexpected behavior as we roll it out.

**Comment by sparhami** [comment]
What is still needed:

- [ ] Layers experiment needs to launch (#15158)
  * Alternatively, we need to manage when the slides load in `amp-base-carousel`
- [ ] `scroll-snap-stop` needs to land for Chrome (https://chromestatus.com/feature/5439846480871424)
  * This just hit the beta channel, should reach stable in ~6 weeks

**Comment by sparhami** [comment]
@jridgewell I heard layers is supposed to launch very soon. Is this something we should wait for or should we start implementing something just for `amp-base-carousel`?

**Comment by sparhami** [comment]
Closing this issue in favor of using amp-carousel 0.2 from lightbox gallery.
